app/assets/javascripts/directives/views/accountMenu.js

```diff
@@ -377,6 +377,17 @@ class AccountMenuCtrl extends PureCtrl {
   }
 
   async performImport(data, password) {
+    if (data.keyParams) {
```

Comment:
`keyParams` is the name of `auth_params` as of 004. In <= 003 it's still called `auth_params`.